test/core/memory_usage/memory_usage_test.cc

```diff
@@ -32,9 +32,13 @@
 #include "test/core/util/port.h"
 #include "test/core/util/subprocess.h"
 
-ABSL_FLAG(int, warmup, 100, "Warmup iterations");
-ABSL_FLAG(int, benchmark, 1000, "Benchmark iterations");
-ABSL_FLAG(bool, minstack, false, "Use minimal stack");
+ABSL_FLAG(std::string, benchmark_name, "call", "Which benchmark to run");
+ABSL_FLAG(int, size, 50000, "Number of channels/calls");
+ABSL_FLAG(std::string, scenario_config, "insecure",
+          "Possible Values: minstack (Use minimal stack), resource_quota, "
```

Comment:
for secure, maybe put "Use SSL credentials on server"?

Reply:
Sounds good! I'll make that change